src/pages/en/core-concepts/astro-pages.md

```diff
@@ -124,6 +124,6 @@ export const get: APIRoute = ({ params, request }) => {
 
 ## Custom 404 Error Page
 
-For a custom 404 error page, you can create a `404.astro` file in `/src/pages`.
+For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`. This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
 
-This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
+> NOTE: The custom 404 error page won't work in your dev environment, but you can access it directly (e.g. at `/404`).
```

Comment:
Now that https://github.com/withastro/astro/pull/3331 has been released we can remove this warning.

```suggestion
```

Reply:
Oh sorry — should I have taken action on this? I'll hit the 'Commit suggestion' button above and see what happens! 😅🤞

Reply:
😅 

This is great, thanks! 

As Maintainers, Chris and I *can* commit it ourselves, but we like to give people the chance to do it themselves. And, the more comfortable you feel doing stuff, the easier it is when you contribute **next time**!  ;)

Reply:
Ah, thanks for the teachable moment! 😄  Did I do that right, or is there something left for me to do? (I'm not that experienced with PRs!)